src/TRoom.cpp

```diff
@@ -59,36 +63,53 @@ QDataStream &operator>>(QDataStream& ds, Qt::PenStyle& value)
     return ds;
 }
 
+static const QColor scDefaultHighlightForeground(QColor(255, 150, 0));
```

Comment:
Some of these are only used twice. I don't think the benefit is worth the difficulty of reading the code they add.

Reply:
At the time I started I wasn't sure of how many times I would need each string, or in this case, colour.

Reply:
I understand, but now would be good to clean them up. They do make code difficult to read and there's actually no benefits to using them like that

Reply:
Will see what I can do...